apps/passport-client/pages/index.tsx

```diff
@@ -75,21 +75,40 @@ class App extends React.Component<object, AppState> {
   lastBackgroundPoll = 0;
   activePollTimout: NodeJS.Timeout | undefined = undefined;
 
+  // If the extension is installed, set up the devTools object
```

Comment:
I don't know what these tools are capable of, or how to use them.  Maybe there should be a link to documentation here to provide context for me and future devs?